frontend/lib/tasks/mnoe_frontend_tasks.rake

```diff
@@ -9,8 +9,8 @@ require 'rake/clean'
 namespace :mnoe do
   namespace :frontend do
     # Default version
-    MNOE_ANGULAR_VERSION = "1.0.x"
-    IMPAC_ANGULAR_VERSION = "1.4.x"
+    MNOE_ANGULAR_VERSION = "master"
+    IMPAC_ANGULAR_VERSION = "1.5.0-rc3"
```

Comment:
@ouranos should this be "v1.5.0-rc3"? When I don't have the v in there I get an abiguous argument error from git